doc/modules/mod_carboncopy.md

```diff
@@ -56,7 +56,14 @@ For an Erlang-based test suite, please see [/esl/ejabberd_tests/blob/master/test
 
 ### Options
 
-* **iqdisc** (default: no_queue)
+#### `modules.mod_carboncopy.iqdisc`
```

Comment:
Should we not explain what it does? I don't feel like this option name is self-explanatory. Maybe we could do that in a more general place as this `iqdisc` can be used in quite a few modules

Reply:
There wasn't any description provided before so I didn't put anything here. In fact this option is described in `Modules.md` and - I don't know why - it's omitted everywhere in the repo.